resources/DK

```diff
@@ -1,74 +1,718 @@
+
+
+
+#------------------------------------------------------------------------------------------------------
+# Danish Mercator ETRS89 lat_0=56 x_0=500000 -> y_0=-3716499.0435458627
+#------------------------------------------------------------------------------------------------------
+<DM> proj=merc lat_ts=56 lon_0=9 x_0=500000 y_0=-3716499.0435458627 ellps=GRS80 no_defs
+<DMRC> DK:DM
+#------------------------------------------------------------------------------------------------------
+<gie>
+operation   init=DK:DM
+tolerance   0.1 mm
+accept      12 55
+expect      687178.31448147167  390074.81929540867
+</gie>
+#------------------------------------------------------------------------------------------------------
+#      echo 12 55 0 0 | cct +init=DK:DM
+#    result: 687178.31448147167  390074.81929540867     0.0  0.0
+#------------------------------------------------------------------------------------------------------
+
 
-<DVR90> +proj=vgridshift +grids=dvr90.gtx
 
-<DKTM1> +proj=etmerc +lat_0=0 +lon_0=9 +k=0.99998 +x_0=200000 +y_0=-5000000 +ellps=GRS80 +units=m +no_defs
-<DKTM2> +proj=etmerc +lat_0=0 +lon_0=10 +k=0.99998 +x_0=400000 +y_0=-5000000 +ellps=GRS80 +units=m +no_defs
-<DKTM3> +proj=etmerc +lat_0=0 +lon_0=11.75 +k=0.99998 +x_0=600000 +y_0=-5000000 +ellps=GRS80 +units=m +no_defs
-<DKTM4> +proj=etmerc +lat_0=0 +lon_0=15 +k=1 +x_0=800000 +y_0=-5000000 +ellps=GRS80 +units=m +no_defs
 
-<DKTM1_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:DKTM1
-<DKTM2_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:DKTM2
-<DKTM3_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:DKTM3
-<DKTM4_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:DKTM4
 
 
-<KP2000B> +proj=etmerc +lat_0=0 +lon_0=15 +k=1 +x_0=900000 +y_0=0 +ellps=GRS80 +units=m +no_defs
-<KP2000J> +proj=etmerc +lat_0=0 +lon_0=9.5 +k=0.99995 +x_0=200000 +y_0=0 +ellps=GRS80 +units=m +no_defs
-<KP2000S> +proj=etmerc +lat_0=0 +lon_0=12 +k=0.99995 +x_0=500000 +y_0=0 +ellps=GRS80 +units=m +no_defs
 
-<KP2000B_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:KP2000B
-<KP2000J_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:KP2000J
-<KP2000S_DVR90> +proj=pipeline +step +init=DK:DVR90 +step +init=DK:KP2000S
+#------------------------------------------------------------------------------------------------------
+#  Legacy systems - formal definitions
+#------------------------------------------------------------------------------------------------------
+#  These formal definitions are for illumination - not for computation.
+#  Valid only for observations already measured in the relevant (legacy)
+#  reference frame.
+#
+#  See below for the much more involved setups needed to go to and from
+#  the legacy frames.
+#------------------------------------------------------------------------------------------------------
+# UTM ED50
+<UTM32_ED50_FORMAL> proj=utm zone=32 ellps=intl units=m no_defs
```

Comment:
This really breaks the core idea of this init-file using ETRS89 as pivot datum. I do appreaciate the need to have a simple version of the legacy system. I'd suggest adding another init file called `ED50` and make that a shared init file for all legacy systems based on ED50 in the Nordic countries. Each country specific init file can then include a mapping to ED50. We can then do something like:

```
proj=pipeline +step +init=DK:ED50 +step +init=ED50:SB
```
Which will transform a ETRS89 geodetic coordinate to the Storebælt system (based on ED50).

Assuming that the existing polynomial mappings allow such a setup.